### PR TITLE
Exe

### PR DESCRIPTION
**Pull request by truetechcode**

### CODE REVIEW

File: lib/tic_tac_toe.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module TicTacToe
+end
+
+require_relative "./tic_tac_toe/core_extensions.rb"
+require_relative "./tic_tac_toe/board.rb"
+require_relative "./tic_tac_toe/cell.rb"
+require_relative "./tic_tac_toe/game.rb"
+require_relative "./tic_tac_toe/player.rb"
+
+class Instruct
```
Comment (maelfosso) [inline lib/tic_tac_toe.rb]: That should be into the executable 
Check that link please https://guides.rubygems.org/make-your-own-gem/#adding-an-executable

File: lib/tic_tac_toe.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module TicTacToe
+end
+
+require_relative "./tic_tac_toe/core_extensions.rb"
+require_relative "./tic_tac_toe/board.rb"
+require_relative "./tic_tac_toe/cell.rb"
+require_relative "./tic_tac_toe/game.rb"
+require_relative "./tic_tac_toe/player.rb"
```
Comment (maelfosso) [inline lib/tic_tac_toe.rb:8]: Up to that line it's ok. I think the rest of the file should go into the executable in *bin/tic_tac_toe..*

File: lib/tic_tac_toe.rb (hunk 1 above)
Comment (truetechcode) [reply lib/tic_tac_toe.rb]: Updated.

File: lib/tic_tac_toe.rb (hunk 2 above)
Comment (truetechcode) [reply lib/tic_tac_toe.rb:8]: Updated

**Comment by maelfosso** [comment]
It's OK

**Pull request merged**